js/layout-javascript/news-feed-code.js

```diff
@@ -65,6 +65,8 @@ function DynamicList(id, data) {
 
   this.data.bookmarksEnabled = _this.data.social.bookmark;
 
+  this.data.areSearchIconsEnabled = this.data.filtersEnabled || this.data.bookmarksEnabled || this.data.sortEnabled;
```

Comment:
Variables or properties are pretty much never named like a question.

```suggestion
  this.data.searchIconsEnabled = this.data.filtersEnabled || this.data.bookmarksEnabled || this.data.sortEnabled;
```

Reply:
Fixed.